de.wolfvollprecht.UberWriter.appdata.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">de.wolfvollprecht.UberWriter.desktop</id>
```

Comment:
Run `appstream-util upgrade` on this file.

Reply:
Done: 0ae56d2bf5d2b95510651ed6460f261db47bfbd6
I was running appstream-util validate-relax, didn't know about this one